apis/logout.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$_SESSION['logged_in']=0;
+$_SESSION['admin_id']=0;
+$_SESSION['email']="";
+$_SESSION['name']="";
+
+session_start();
```

Comment:
session_start() should be in the beginning.the  upper statements wont work without the session_start